uaclient/config.py

```diff
@@ -100,6 +104,14 @@
 
 event = event_logger.get_event_logger()
 
+# needed for solving mypy errors dealing with _lru_cache_wrapper
+# Found at https://github.com/python/mypy/issues/5858#issuecomment-454144705
+S = TypeVar("S", bound=str)
+
+
+def str_cache(func: Callable[..., S]) -> S:
```

Comment:
@lucasmoura @renanrodrigo  I got this workaround from [here](https://github.com/python/mypy/issues/5858). Mypy and lru_cache do not play well.